src/components/App.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
+// this is commented out to avoid the linter error 'is defined but never used'
+// import calculate from '../logic/calculate';
```

Comment:
[REQUIRED] calculate.js should be imported in the App component

Reply:
I know but if I leave it there will be a linter error and the checks will not pass.
Should I disable Eslint on that line?